hubblestack/extmods/returners/splunk_nebula_return.py

```diff
@@ -49,11 +49,12 @@
 import json
 import time
 from datetime import datetime
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Let's not disable these warnings globally. If the connection is insecure, we want to know about it. I like the ability to disable verification, but the warning should stay. Or there should be a configurable parameter to disable warnings, and the parameter should default to showing the warnings.